### PR TITLE
🎉 fix Google Adwords CI & publish new version

### LINKED ISSUE #2929: fix Google Adwords CI  & publish new version
## Description
The Google Adwords CI is [failing](https://status-api.airbyte.io/tests/summary/source-google-adwords-singer/). I'm pretty sure nothing is wrong with the connector but it seems like we may have no data in the underlying source. 

Once this has been fixed, we should bump & publish a new version to pick up the changes from https://github.com/airbytehq/airbyte/pull/2918

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368101988801) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by po3na4skld**
## What
It solves #2929 

## How
Changing the states to SAT 

## Recommended reading order
1. configured_catalog.json

## Pre-merge Checklist
Expand the checklist which is relevant for this PR. 

<details><summary> <strong> Connector checklist </strong> </summary>
<p>

- [x] Issue acceptance criteria met
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [x] Unit & integration tests added as appropriate (and are passing)
- [ ] `/test connector=connectors/<name>` command as documented [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-an-existing-connector) is passing. 
- [ ] Code reviews completed
- [ ] Credentials added to Github CI if needed and not already present. [instructions for injecting secrets into CI](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#using-credentials-in-ci). 
- [ ] Build is successful
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] No major blockers
- [ ] PR merged into master branch
- [ ] Associated tickets have been closed & stakeholders notified
</p>
</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "sample_files/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "sample_files/configured_catalog.json"
```
Comment (keu) [inline airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml:14]: lets move `invalid_config` and `configured_catalog` to `integration_tests` folder

File: airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      validate_output_from_all_streams: yes
+  full_refresh:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "sample_files/configured_catalog.json"
```
Comment (keu) [inline airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml:18]: what about the incremental test? was it disabled? can leave a comment here explaining why

File: airbyte-integrations/connectors/source-google-adwords-singer/build.gradle [hunk 3]
```diff
@@ -21,10 +21,3 @@ task('installSingerTap', type: PythonTask) {
     command = "install tap-adwords==1.12.0"
```
Comment (keu) [inline airbyte-integrations/connectors/source-google-adwords-singer/build.gradle:21]: lets move `tap-adwords==1.12.0` to `setup.py` and remove this task (`installSingerTap`)

File: airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml (hunk 2 above)
Comment (po3na4skld) [reply airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml]: Yeah they were disabled, I added comment why.

File: airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml (hunk 1 above)
Comment (po3na4skld) [reply airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml]: OK

File: airbyte-integrations/connectors/source-google-adwords-singer/build.gradle (hunk 3 above)
Comment (po3na4skld) [reply airbyte-integrations/connectors/source-google-adwords-singer/build.gradle]: OK

Comment (po3na4skld) [reply airbyte-integrations/connectors/source-google-adwords-singer/build.gradle]: @keu I can't run tests with this in setup.py. Also it was specified in Dokerfile before me. The problem is in pip install requirements.txt it fails with timeout error

**Comment by vitaliizazmic** [comment]
@po3na4skld  as I see, you have only enabled SAT for the Source. Could you please clarify issue and how it helped fix bug? Also could you add CHANGELOG.md and don't forget to bump version.

**Review by vitaliizazmic — COMMENTED** [review]
@po3na4skld as I see, you have only enabled SAT for the Source. Could you please clarify issue and how it helped fix bug? Also could you add CHANGELOG.md and don't forget to bump version and publish.

**Comment by keu** [comment]
@po3na4skld  @vitaliizazmic CHANGELOG.md has being moved to <connector>.md file

File: airbyte-integrations/connectors/source-google-adwords-singer/build.gradle (hunk 3 above)
Comment (keu) [reply airbyte-integrations/connectors/source-google-adwords-singer/build.gradle]: ok, lets keep it as it is for now

File: airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+      validate_output_from_all_streams: yes
+  full_refresh:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+
+# incremental tests are disabled due to unsupported state structure in current tests
+# required structure: {stream_name: cursor_value} given {‘bookmarks’: {stream_name: cursor_value}}
```
Comment (keu) [inline airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml:21]: missing new line

**Review by keu — APPROVED** [review]
lgtm, just small fixes

File: airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml (hunk 4 above)
Comment (keu) [reply airbyte-integrations/connectors/source-google-adwords-singer/acceptance-test-config.yml:21]:
```suggestion
# required structure: {stream_name: cursor_value} given {‘bookmarks’: {stream_name: cursor_value}}

```

**Comment by po3na4skld** [comment]
> @po3na4skld as I see, you have only enabled SAT for the Source. Could you please clarify issue and how it helped fix bug? Also could you add CHANGELOG.md and don't forget to bump version and publish.

@vitaliizazmic I added them because previous tests were failing due to error: Expected the first incremental sync to produce records. As we have all streams with full refresh, we decided to remove incremental testings as it is not supported for this singer for now.

**Review by vitaliizazmic — APPROVED** [review]

**Review by tuliren — APPROVED** [review]

File: docs/integrations/sources/google-adwords.md [hunk 5]
```diff
@@ -60,3 +60,9 @@ When you apply for a token, you need to mention:
 
 If for any reason the request gets denied, let us know and we will be able to unblock you.
 
+
+## Changelog
+
+| Version | Date       | Pull Request | Subject |
+| :------ | :--------  | :-----       | :------ |
+| 0.1.2  | 2021-06-25 | [4205](https://github.com/airbytehq/airbyte/pull/4205) | Set up CDK SAT tests. Incremental tests are disabled due to unsupported state structure in current tests: required structure: {stream_name: cursor_value} given {‘bookmarks’: {stream_name: cursor_value}} |
```
Comment (sherifnada) [inline docs/integrations/sources/google-adwords.md:68]:
```suggestion
| 0.1.2  | 2021-06-25 | [4205](https://github.com/airbytehq/airbyte/pull/4205) | Set up CDK SAT tests. |
```

**Review by sherifnada — APPROVED** [review]
@po3na4skld can you update the PR description to explain what changes this PR made? Imagine someone reading the commit log -- "fix the connector" doesn't tell me what this PR is doing 😜

File: docs/integrations/sources/google-adwords.md (hunk 5 above)
Comment (keu) [reply docs/integrations/sources/google-adwords.md:68]: @po3na4skld the log message is wrong, you bump version to 0.2.5, but logs say about 0.1.2.
Also why did you bumped 0.2.3 to 0.2.5, where is 0.2.4?

Comment (sherifnada) [reply docs/integrations/sources/google-adwords.md:68]: good catch eguene. 

skipping the version isn't a big problem, but we should definitely list the correct version in the changelog

**Pull request merged**